client/src/js/constants/bhConstants.js

```diff
@@ -13,6 +13,11 @@ function constantConfig() {
     actions : {
       CAN_EDIT_ROLES : 1,
       CAN_UNPOST_TRANSACTIONS : 2,
+      DELETE_CASH_PAYMENT : 3,
+      DELETE_INVOICE : 4,
+      DELETE_PURCHASE_ORDER : 5,
+      DELETE_STOCK_MOVEMENT : 6,
+      DELETE_VOUCHER : 7,
```

Comment:
This is a good thought, but at this point there aren't enough people smashing in permissions to make it worth coming up with a system.  This would make a lot more sense for our `unit` table, actually.  The probably is that we'd need to deal with legacy installations...